package.json

```
@@ -2,6 +2,9 @@
   "version": "1.37.2",
   "title": "shop-standards",
   "description": "Standard refinements for e-commerce websites.",
+  "scripts": {
+    "build": "gulp build --concat=false --production"
```

@gyopiazza This is based on a very old version of gulp-task-collection. In version 3.0 we added some improvements so it might make sense to upgrade it here. 

If you upgrade gulp-task-collection then you can probably change these build scripts up a bit

```suggestion
    "watch": "gulp",
    "build": "gulp build:production"
```